apps/graphql/src/apps/booking/types/inputs/HoldBags.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import { GraphQLObjectType, GraphQLInt } from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'HoldBags',
+  description: 'Number of hold bags for a passenger',
+  fields: {
+    first: {
```

Comment:
Could you add some description for the fields as well, I don't understand how these fields work.

Reply:
@tbergq this `HoldBags` field, in particular, is a little bit problematic. It is quite confusing on the documentation: https://tequila.kiwi.com/portal/resources/booking_api and also has fields that are integers (changed it to `first`, `second`) and that's not allowed, but I have a workaround for that... (just sanitize them before sending them to the data loader)

Also, due to that field, we have to pass the flight ID to the query...

Reply:
Yes, but still, some description in the field would be nice, when I am looking on `graphiql`, I would want to know what is this field.

Reply:
Done!